WordPress/src/main/java/org/wordpress/android/ui/posts/BasicFragmentDialog.kt

```diff
@@ -37,7 +37,7 @@ class BasicFragmentDialog : AppCompatDialogFragment() {
 
     fun initialize(
         tag: String,
-        title: String,
+        title: String? = null,
```

Comment:
I might be a little paranoid, but isn't it a little risky to have a default value for this parameter due to ordering? We have 2 other parameters with default values as the last 2 parameters and now one in the middle. When I don't use the parameter names while calling it (which is always the case if it's called from Java) I can't actually just pass 3 parameters to this. I believe that's because the JVM thinks that the second parameter is for the `title` even though it's for the `message` since `title` is optional.

This might be just me being paranoid, but I'd personally feel more comfortable to make this a nullable parameter without any default values.

Reply:
I haven't realized that. I completely agree with you. Fixed in #[0fe922d](https://github.com/wordpress-mobile/WordPress-Android/pull/9187/commits/0fe922d705373909459d6c0bfe4a722dda21c95d).